python/lsst/ts/ess/controller/device/rpi_serial_hat.py

```diff
@@ -61,7 +61,7 @@ def __init__(
             log=log,
         )
         try:
-            self.ser = serial.Serial(port=device_id, baudrate=19600)
+            self.ser = serial.Serial(port=device_id, baudrate=19200)
```

Comment:
Given the need to fix this in two places, please consider a constant (or set/dict/enum of constants).

Reply:
Good idea. Done.